application/migrations/20200130145100_add_org_domain_authconfig.up.fizz

```diff
@@ -0,0 +1,2 @@
+add_column("organization_domains", "auth_type", "string", {null: true})
```

Comment:
Do these need to be nullable or just default to an empty string? Your check in `GetAuthProvider` is against an empty string (not `AuthType.Valid`) anyway.

Reply:
Interesting, good point, I'm not really sure. I assume most of the time these fields wont be used, so I just went with null, but if the other is better feel free to change it.